src/device-registry/utils/create-know-your-air.js

```diff
@@ -455,10 +455,13 @@ const createKnowYourAir = {
 
           if (responseFromListProgress.data.length == 0) {
```

Comment:
@BenjaminSsempala , in future PRs, use the tripple (===) instead of the double (==). You can read more about the reason behind this on the internet.

Reply:
Thank you @Baalmart I'll update this in any followup PRs